python-threatexchange/threatexchange/state.py

```diff
@@ -148,12 +153,12 @@ def _read_state(
 
             logging.debug("Loaded %s with %d records", collab_name, len(delta.updates))
             return delta
-        except Exception:
+        except Exception as exc:
             logging.exception("Failed to read state for %s", collab_name)
-            raise CommandError(
+            raise TXStateException(
                 f"Failed to read state for {collab_name}. "
                 "You might have to delete it with `threatexchange fetch --clear`"
-            )
+            ) from exc
```

Comment:
I think this is captured by default in modern python, and might explicitly hint you don't want the full stack printed (or maybe that's `from None`

Reply:
Oh ok, I was following a linter recommendation but it may have been an out of date rule